File: src/main/kotlin/com/yosep/server/infrastructure/redis/component/RedisPrefixHelper.java
package com.yosep.server.infrastructure.redis.component;

public class RedisPrefixHelper {

    public static String getRedisKey(String prefix, String... params) {
        StringBuilder sb = new StringBuilder(prefix);

        for (String param : params) {
            sb.append(":");
            sb.append(param);
        }

        return sb.toString();
    }
}

File: src/main/kotlin/com/yosep/server/common/util/RedisPrefixHelper.kt
package com.yosep.server.common.util

object RedisPrefixHelper {
    fun getRedisKey(prefix: String, vararg params: String?): String {
        val sb = StringBuilder(prefix)

        for (param in params) {
            sb.append(":")
            sb.append(param)
        }

        return sb.toString()
    }
}